docs/source/onnxruntime/package_reference/modeling_ort.mdx

```diff
@@ -66,3 +66,19 @@ specific language governing permissions and limitations under the License.
 ## ORTModelForVision2Seq
 
 [[autodoc]] onnxruntime.ORTModelForVision2Seq
+
+## ORTModelForAudioClassification
```

Comment:
For now it was ordered alphabetically, but maybe it makes sense to have categories by modality (text, audio, image)

Reply:
Added